lib/unified_search_builder.rb

```diff
@@ -397,4 +398,14 @@ def devolved_org_boost
     }
   end
 
+  def entities_boost
+    {
+      filter: { terms: { entities: find_entities} },
+      boost: 20
+    } unless find_entities.empty?
+  end
+
+  def find_entities
+    @entities ||= @entity_extractor.call(@query) || []
```

Comment:
The 'error swallowing' one used in development will return nil if it can't connect to the entity extractor on its second (or subsequent) attempts.